### PR TITLE
Feat/scatter gather

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR includes steps for scattering the input vcf for vep, and merging the output back into one vcf for the tools downstream. 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by jemten — COMMENTED** [review]
Looks good. Se my small comments

File: conf/modules.config [hunk 1]
```diff
@@ -711,7 +729,21 @@ process {
         ]
     }
 
+    withName: '.*ANNOTATE_SNVS:GATK4_SELECTVARIANTS' {
+        cpus = 1
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply conf/modules.config]: Right! Removed it 👍🏻

File: conf/modules.config [hunk 2]
```diff
@@ -711,7 +729,21 @@ process {
...
+        memory = 60.GB
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply conf/modules.config]: Removed it from this config

File: subworkflows/local/scatter_genome.nf [hunk 3]
```diff
@@ -0,0 +1,24 @@
+include { BUILD_BED            } from '../../modules/local/create_bed_from_fai'
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply subworkflows/local/scatter_genome.nf:5]: Done 👍🏻

**Review by jemten — APPROVED** [review]
✅

**Pull request merged**